docs/site_navigation/redirects.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Redirects
+parent: Site Navigation
+nav_order: 3
+---
+
+# Redirects
+
+The URL redirects are managed in **My Site > Redirects**, both Creators and Developers have access.
+
+## URLs that can't be redirected
+
+Redirects can't be set up **to** or **from** URL paths that start with:
```

Comment:
I think you can always redirect *to* the urls, you're just not able to redirect from them.

As Lizzy says above I think you can now redirect a url with /products/ or /blog/ but with the caveat that if a page/product/blog exists with that url it must be unpublished for the redirect to work.

Reply:
I've completed the information in the next commit